airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/operations/RedshiftS3StagingSqlOperations.java

```diff
@@ -59,6 +59,7 @@ public RedshiftS3StagingSqlOperations(final NamingConventionTransformer nameTran
    *
    */
   @Override
+  @Deprecated
```

Comment:
is the superclass method also deprecated? or just the redshift implementation of it

and nit - would be nice to have `@Deprecated("Becuase xyz, use foo instead")`

Reply:
Looked into this today and it is not used anywhere meaningful, removing it
![image](https://github.com/airbytehq/airbyte/assets/4081089/5bccb06e-41fd-4149-8a66-a6d42a85d74c)